src/java/PC.java

```diff
@@ -0,0 +1,64 @@
+package java;
+import java.util.Objects;
+import java.util.Scanner;
+public class PC {
+    private int attempt;
+    private boolean cpu;
+    private boolean ram;
+    private boolean hdd;
+    private boolean isOn;
+    private boolean burn;
+    private int resource;
+    private String activation = "On";
+
+    public PC(int resource){
+        this.resource = resource;
+    }
+    public void Description(){
+        System.out.print("[" +cpu+ "," +ram+ "," +hdd+ "," +resource+ "]");
+        System.out.println();
+       /* System.out.print(ram + ",");
+        System.out.print(hdd + ",");
+        System.out.println(resource + "]");*/
+    }
+    private void On(){
+        this.cpu = true;
+        this.ram = true;
+        this.hdd = true;
+    }
+    private void Off(){
+        this.cpu = false;
+        this.ram = false;
+        this.hdd = false;
+    }
+    public void Check(){
```

Comment:
@ZsamatkhanoV  Между методами 1 пустая строка должна быть
```suggestion
    }
    
    private void On(){
        this.cpu = true;
        this.ram = true;
        this.hdd = true;
    }
    
    private void Off(){
        this.cpu = false;
        this.ram = false;
        this.hdd = false;
    }
    
    public void Check(){
```